src/wrappers.jl

```diff
@@ -0,0 +1,241 @@
+
+export Point, EmptyPoint, LineString, Polygon, Triangle, MultiPoint, MultiCurve, MultiPolygon, TIN, Collection, Feature, FeatureCollection
+
+# Implement interface
+
+"""
+    WrapperGeometry
```

Comment:
```suggestion
    abstract type WrapperGeometry{Z,M,T}
```

Would be good to add to the docstring that Z and M are boolean type parameters that indicate if Z and M dimensions are present, and T is the geometry type.